### PR TITLE
[Ascend] Add LLaVA_1.5_7b Pretrain and Fintune for Ascend

### PR DESCRIPTION
**Pull request by zbj-china**
Add LLaVA_1.5_7b Pretrain and Fintune for Ascend

### CODE REVIEW

File: training/ascend/llava1.5_7b-deepspeed-torch/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+| 硬件设备简称 | Ascend 800T A2    |                                    |
+| 硬件存储使用 | mem,见“性能指标”           | 通常称为“显存”,单位为GiB           |
+| 计算使用率 | MFU,见“性能指标”           | 参见PaLM论文定义 |
+| **吞吐量**   | **token/p/s,见“性能指标”** | 平均单卡每秒处理的token数          |
+| MMMU（val）结果           | acc(推理/验证)   | MMMU（val）回答准确率                   |
+* 性能指标
+
+| 配置                |  fix_hp           | token/p/s | loss | mem       |acc(MMMU) |MFU       |
+| ------------------- | ---------------- | ------ | ------- | --------- | --------- |--------- |
+| A800单机8卡（1x8）（pretrain）  |  74/81  | 3448 | 0.0272 | / | - | 15.47% |
```
Comment (shh2000) [inline training/ascend/llava1.5_7b-deepspeed-torch/README.md:57]: you should use 910B's results, not A800

File: training/ascend/llava1.5_7b-deepspeed-torch/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+| 硬件设备简称 | Ascend 800T A2    |                                    |
+| 硬件存储使用 | mem,见“性能指标”           | 通常称为“显存”,单位为GiB           |
+| 计算使用率 | MFU,见“性能指标”           | 参见PaLM论文定义 |
+| **吞吐量**   | **token/p/s,见“性能指标”** | 平均单卡每秒处理的token数          |
+| MMMU（val）结果           | acc(推理/验证)   | MMMU（val）回答准确率                   |
+* 性能指标
+
+| 配置                |  fix_hp           | token/p/s | loss | mem       |acc(MMMU) |MFU       |
+| ------------------- | ---------------- | ------ | ------- | --------- | --------- |--------- |
+| Atlas 800T A2单机8卡（1x8）（pretrain）  |  74/81  | 3448 | 0.0272 | / | - | 15.47% |
```
Comment (shh2000) [inline training/ascend/llava1.5_7b-deepspeed-torch/README.md:57]: fix_hp与mem信息填反

File: training/ascend/llava1.5_7b-deepspeed-torch/config/config_Atlas800TA2x1x8.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    os.system("mv modeling_llama.py " + modeling_llama_path +";")
+    
+# useing torch_npu
+os.system("cp ./train_mem.py ../../../benchmarks/llava1.5_7b/deepspeed-torch/train/")
+
+os.chdir(cwd)
+
+
+# Common arguments
+theoryflops = 312000000000000.0
```
Comment (shh2000) [inline training/ascend/llava1.5_7b-deepspeed-torch/config/config_Atlas800TA2x1x8.py:39]: should use atlas800's theoryflops

**Review by shh2000 — APPROVED** [review]

File: training/ascend/llava1.5_7b-deepspeed-torch/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+   - 实施细节：ZeRO-DP Pretrain-stage:O2 Finetune-stage:O3
+
+- ##### 优化策略
+
+   - flash attention 2
+
+### 运行情况
+
+* 输入批尺寸
+  1. local_batchsize(micro_batchsize)，简写为LBS，即实际进入模型的张量批尺寸，为config_A800x1x8.py中所写，在本case中pretrain阶段为32，finetune阶段为16
```
Comment (KerwinKai) [inline training/ascend/llava1.5_7b-deepspeed-torch/README.md:29]: 需要在文档中将config_A800x1x8.py修改为config_Atlas800TA2x1x8.py

**Comment by KerwinKai** [comment]
请在该文件中补充启动配置：https://github.com/FlagOpen/FlagPerf/blob/main/training/run_benchmarks/config/test_conf.py

**Review by KerwinKai — APPROVED** [review]

**Pull request merged**